cluster/manifests/skipper/deployment.yaml

```diff
@@ -128,6 +128,41 @@ spec:
           - name: filters
             mountPath: /etc/config/default-filters
 {{ end }}
+{{ if eq .ConfigItems.tokeninfo_url "http://127.0.0.1:9021/oauth2/tokeninfo" }}
+      - name: tokeninfo
+        image: registry.opensource.zalan.do/foundation/platform-iam-tokeninfo:master-1
+        ports:
+        - containerPort: 9021
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 9021
+          periodSeconds: 3
+          failureThreshold: 2
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 64Mi
+          requests:
+            cpu: 1000m
+            memory: 64Mi
```

Comment:
Should we increase the memory requirements a bit, since we're already provisioning a ton of CPU?

Reply:
I just used values from their configuration, wdyt?
Do you have any suggestion?
I think the skipper auth connection pool does not allow connection spikes, so I thought this could be fine.

Reply:
Why not? A CPU reservation of `1` already reserves 25% of our default instance, so the memory would be unused anyway.

Reply:
default now 256Mi but configurable via CR